test/e2e/networking.go

```diff
@@ -241,6 +208,50 @@ var _ = Describe("Networking", func() {
 		Expect(string(body)).To(Equal("pass"))
 	})
 
+	// Marked with [Skipped] to skip the tests by default.  These
```

Comment:
these tests seem generic/useful enough that they could be part of the default suite, though possibly tag as `[Flaky]` to start while we verify they're stable.